testsuite/tests/asmcomp/func_sections.run

```diff
@@ -7,4 +7,4 @@ ${program}
 
 # now check the assembly file produced during compilation
 asm=${test_build_directory}/func_sections.s
-grep "\.section \.text\.caml\.camlFunc_sections\\$" "$asm" | wc -l | tr -d ' ' | sed '/^$/d'
+grep "\.section \.text\.caml\.camlFunc_sections." "$asm" | wc -l | tr -d ' ' | sed '/^$/d'
```

Comment:
Using just `.` matches any character. I'm not sure this is what you had in mind. Maybe something like that instead?
```suggestion
grep "\.section \.text\.caml\.camlFunc_sections[.$]" "$asm" | wc -l | tr -d ' ' | sed '/^$/d'
```

Reply:
That was a quick fix, but yes I agree it is better to narrow this `grep`.